src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -212,36 +211,48 @@ void merge_config_with(ov::AnyMap& lhs, const ov::AnyMap& rhs) {
     }
 }
 
-ov::AnyMap get_default_prefill_config(const std::shared_ptr<ov::Model>& model) {
+ov::AnyMap get_baseline_common_config() {
     ov::AnyMap config = {
-        { "NPU_USE_NPUW", "YES" },
+        { "NPU_COMPILATION_MODE_PARAMS", "compute-layers-with-higher-precision=Sqrt,Power,ReduceMean,Add_RMSNorm" },
+        { "NPU_USE_NPUW",  "YES" },
         { "NPUW_FOLD", "YES" },
         { "NPUW_DCOFF_TYPE", "f16" },
-        { "NPUW_DCOFF_SCALE",  "YES" },
-        { "NPUW_WEIGHTS_BANK",  "shared" },
-        { "NPU_COMPILATION_MODE_PARAMS", "compute-layers-with-higher-precision=Sqrt,Power,ReduceMean,Add" },
-        { "NPUW_ONLINE_AVOID", "P:RMSNorm/NPU" }
+        { "NPUW_DCOFF_SCALE", "YES"},
+        { "NPUW_WEIGHTS_BANK", "shared" }
     };
-    enable_npuw_dq_if_allowed(config, model);
     return config;
 }
 
-ov::AnyMap get_default_generate_config(const std::shared_ptr<ov::Model>& model) {
-    ov::AnyMap config = {
-        { "NPU_USE_NPUW", "YES" },
-        { "NPUW_FOLD", "YES" },
-        { "NPUW_DCOFF_TYPE", "f16" },
-        { "NPUW_DCOFF_SCALE", "YES" },
-        { "NPU_COMPILATION_MODE_PARAMS", "compute-layers-with-higher-precision=Sqrt,Power,ReduceMean,Add" },
-        { "NPUW_PARALLEL_COMPILE", "YES" },
-        { "NPUW_FUNCALL_ASYNC", "YES" },
-        { "NPUW_WEIGHTS_BANK",  "shared" },
-        { "NPUW_ONLINE_AVOID", "P:RMSNorm/NPU" }
-    };
+struct NPUDesc {
+    bool support_max_mem_alloc_size;
+};
+
+ov::AnyMap get_default_common_config(const std::shared_ptr<ov::Model>& model,
+                                     const std::optional<NPUDesc>& desc) {
+    auto config = get_baseline_common_config();
+    if (desc.has_value() && desc->support_max_mem_alloc_size) {
+        config.emplace("NPUW_PMM", "NO");
```

Comment:
Disabled PMM can be in the base (common) config I believe

Reply:
DOne